tests/integration.rs

```diff
@@ -590,6 +590,17 @@ mod tests {
                      self.disk_config.disk(DiskType::OperatingSystem).unwrap().as_str(),
                      self.disk_config.disk(DiskType::CloudInit).unwrap().as_str(),
             }
+            #[cfg(all(feature = "mshv", target_arch = "x86_64"))]
+            format! {"{{\"cpus\":{{\"boot_vcpus\":{},\"max_vcpus\":{}}},\"kernel\":{{\"path\":\"{}\"}},\"cmdline\":{{\"args\": \"{}\"}},\"net\":[{{\"ip\":\"{}\", \"mask\":\"255.255.255.0\", \"mac\":\"{}\"}}], \"disks\":[{{\"path\":\"{}\"}}, {{\"path\":\"{}\"}}]}}",
+                     cpu_count,
+                     cpu_count,
+                     direct_kernel_boot_path().to_str().unwrap(),
+                     DIRECT_KERNEL_BOOT_CMDLINE,
+                     self.network.host_ip,
+                     self.network.guest_mac,
+                     self.disk_config.disk(DiskType::OperatingSystem).unwrap().as_str(),
+                     self.disk_config.disk(DiskType::CloudInit).unwrap().as_str(),
+            }
```

Comment:
If the only difference is the kernel parameter, could you please factorize it behind a variable and pass this variable to the `format!()` function on line 587?

Reply:
Does the latest change make sense?

Reply:
Yes thanks